src/Symfony/Bridge/Monolog/Processor/DebugProcessor.php

```diff
@@ -45,6 +66,15 @@ public function __invoke(array $record)
      */
     public function getLogs()
     {
+        if (func_num_args() === 1) {
+            $requestHash = func_get_arg(0);
+            if (isset($this->recordsRequest[$requestHash]['records'])
+                && $requestHash !== $this->masterRequestHash
```

Comment:
We usually don't split these long `if()` conditions, so this should be one line.